src/core/tests/Test_YumPackageManager.py

```diff
@@ -597,6 +600,15 @@ def test_update_image_default_patch_mode_raises_exception(self):
         self.runtime.env_layer.file_system.write_with_retry = self.mock_write_with_retry_raise_exception
         self.assertRaises(Exception, package_manager.update_os_patch_configuration_sub_setting)
 
+    def test_is_reboot_pending_return_true_when_exception_raised(self):
+        package_manager = self.container.get('package_manager')
+        backup_do_process_require_restart = package_manager.do_processes_require_restart
+        package_manager.do_processes_require_restart = self.mock_do_processes_require_restart_raise_exception
+
```

Comment:
very minor: It's okay if we don't have an extra line break for smaller test sections. Eg: there's just one assert here, so I wouldn't  add the empty line on 607 and 609

Reply:
Keeping this as is to make it easier to separate the setup/ test /clean up stages.